### PR TITLE
fix #9325

### LINKED ISSUE #9325: Cross-App UI changes (Blockframes, Archipel Content, Archipel Market)
While working on the new Waterfall application we are would like to introduce some UI updates that are cross-app. 

- [x] Verify the logic of the color chart and see the naming for the new BG color for the light-theme (@2amarkina) ; 
- [x] Apply new background color for the light theme #FAFBFF (to be tested with design in life to check if all good) ; 
- [x] Remove the logo in the footer (Dashboard, Marketplace) ; 

For the card-surfaces on the Dashboards (see example below):  

- [x] make the entire the card-surface clickable ; 
- [x] add shadowing effect on hover for the light theme on hover 
`.surface.ng-star-inserted {
   filter: drop-shadow(0px 5px 5px rgba(8, 11, 15, 0.20));
} 
` where RGB is a variation of #080b0f (Light Theme / Foreground 900) ; 

- [x] add shadowing effect on hover for the dark theme on hover 
`.surface.ng-star-inserted {
    filter: drop-shadow(0px 5px 5px rgba(199, 207, 242, 0.20));
}
`
where RGB is a variation of #C7CFF2 (Dark Theme / Foreground 800) ;

- [x] change the cursor to pointer to emphasize the possibility to click for the user.

### PR DESCRIPTION
**Pull request by pldespaigne**
implements changes requested in #9325

### CODE REVIEW

File: libs/ui/src/lib/layout/footer/footer.component.html [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <mat-divider></mat-divider>
```
Comment (phpgeekfr) [inline libs/ui/src/lib/layout/footer/footer.component.html:1]: I think you can remove `AppLogoModule` in libs/ui/src/lib/layout/footer/footer.module.ts

File: styles/themes/blockframes.scss [hunk 2]
```diff
@@ -76,7 +76,7 @@ $light-theme: (
     ),
   ),
   background: (
-    900: #f4f4f4,
+    900: #fafbff,
```
Comment (phpgeekfr) [inline styles/themes/blockframes.scss:79]: you might also want  to change this on `styles/themes/screen-daily.scss` this is a theme we use during berlin festival

**Review by phpgeekfr — APPROVED** [review]

File: libs/ui/src/lib/dashboard/pages/home/home.component.html [hunk 3]
```diff
@@ -39,19 +39,19 @@ <h1>No title to display.</h1>
       <article fxLayout="row" fxLayout.lt-md="column" fxLayoutAlign="center" fxLayoutGap="24px">
         <!-- Should only be shown for noAcceptedMovies && !noMovies -->
         <ng-container *ngIf="titles | hasAppStatus:'draft'">
-          <div class="surface" fxLayout="column" fxLayoutalign="center center">
+          <a routerLink="../title" class="surface" fxLayout="column" fxLayoutalign="center center">
```
Comment (phpgeekfr) [inline libs/ui/src/lib/dashboard/pages/home/home.component.html:42]: I did not think about it at first, but I think this have to be done here also `apps/festival/festival/src/app/dashboard/home/home.component.html`

Comment (pldespaigne) [reply libs/ui/src/lib/dashboard/pages/home/home.component.html:42]: fixed by #9332

**Pull request merged**